### ISSUE TITLE
Squashing some compile warnings?

### ISSUE THREAD

**Issue by Martii** [mongodb-js/kerberos#71]
Hello,

Thanks for having this project available. 

So onto business... we're using *kerberos*@0.0.23 and older for a few years now and I was wondering if it would be possible to squash at least three of the warnings with the following commit on a fork at https://github.com/Martii/kerberos/commit/2eb643ecf544ae42b4a0e2631a20074e1688a3ea . I'm a bit rusty on my C++ but capable of finding at least one solution when needed. :)

Ref(s):
* Issues with this indirectly mentioned over the years at https://github.com/mongodb-js/kerberos/search?q=server_conf_flags&type=Issues
* gcc here under Linux not clang under OS X nor Microsoft Visual C++/Studio.

Console **with** the commit referenced:
``` sh-session
$ npm i

> kerberos@0.0.23 install ~/repo/git/OpenUserJS.org/martii/OpenUserJS.org/node_modules/kerberos
> (node-gyp rebuild) || (exit 0)

make: Entering directory '~/repo/git/OpenUserJS.org/martii/OpenUserJS.org/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
  CXX(target) Release/obj.target/kerberos/lib/worker.o
  CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
../lib/kerberosgss.c: In function ‘authenticate_gss_client_wrap’:
../lib/kerberosgss.c:448:19: warning: variable ‘server_conf_flags’ set but not used [-Wunused-but-set-variable]
   char buf[4096], server_conf_flags;
                   ^
  CC(target) Release/obj.target/kerberos/lib/base64.o
  CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
  SOLINK_MODULE(target) Release/obj.target/kerberos.node
  COPY Release/kerberos.node
make: Leaving directory '~/repo/git/OpenUserJS.org/martii/OpenUserJS.org/node_modules/kerberos/build
...
```

The last remaining "warning" is that `PRINTFS` isn't defined so perhaps an output to the cross-platform equivalent of `/dev/nul`, or perhaps just invalidating the value, or something else *([See here](https://stackoverflow.com/questions/3599160/unused-parameter-warnings-in-c) perhaps)*?

I'm aware that these are "okay to ignore" but it's quite disconcerting to see these anytime a fresh app is `npm i`'d... and we were hoping to have the output managed as professionally as possible.

Thanks for the feedback.

**Comment by Martii** [comment]
There's too many choices to choose over on stackoverflow... so I really don't know what would be accepted here for the unused-but-set-variable warning.

Did get the following result back then but not tested cross-platform *(Don't have MSVC either)*. See https://github.com/Martii/kerberos/commit/e260e833e3bcd407f9ed3dfdc7145e803466aa7c

**Comment by mbroadst** [comment]
fixed in 048479d8c0bbd3f9ef63dbcdfac6e9ff02147b3b
